arches_lingo/settings.py

```diff
@@ -145,6 +144,7 @@
     "corsheaders",
     "oauth2_provider",
     "django_celery_results",
+    "arches_references",
```

Comment:
Don't we want this above arches since it's an arches application?

Reply:
it doesn't matter, as long as `arches.app` is at the bottom

Reply:
I thought that was only true for templates, not css/img discovery.

Reply:
just testing with an image and the results are as you'd expect ( project -> apps -> core )

Reply:
Okay, I'm with you now. I forgot how specialized we made `ArchesApplicationsStaticFilesFinder`, which even if there's an `"arches"` in the middle of INSTALLED_APPS, there won't be anything under `arches/media`, so this should be A-OK.

Reply:
@jacobtylerwalls @chrabyrd back from the future - `arches_references` has to go above `arches` because there are commands in `arches_references/packages.py` that need to be included in the packages options.  I'll fix as part of #53